rts/lra/proxy/test/pom.xml

```diff
@@ -44,6 +44,10 @@
       <artifactId>resteasy-client</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
I needed to add jakarta.ws.rs-api dependency with scope 'compile' (default one) otherwise the compilation fails after removing scope from dependencies' bom.

Reply:
+1